### PR TITLE
PIE-3147: Added vip prefix for Pendo tracking

### PR DESCRIPTION
**Pull request by kat3samsin**
## Description

Added a `vip` prefix for VIP users for better Pendo tracking.

## Steps to Test
Make sure VIP is prefixed to VIP users
1. Check out PR.
2. Run `npm run build`
3. Login using VIP account `./dist/bin/vip.js login`
4. Run `./dist/bin/vip-whoami.js -d`
5. Verify that debug message `pendo send()` contains updated `vip-<id>`

Make sure there is no VIP prefixed to non-VIP users
1. Login using non VIP account `./dist/bin/vip.js login`
2. Run `./dist/bin/vip-whoami.js -d`
3. Verify that debug message `pendo send()` does  NOT contain `vip-` prefix

### CODE REVIEW

**Review by sjinks — APPROVED** [review]

**Comment by andrea-sdl** [comment]
@sjinks we might not need this PR as we might end up doing the `vip` prefixing in the dedicated `/pendo` endpoint (leaving the VIP CLI code cleaner).
Let's wait for @kat3samsin before merging. Thanks.

**Comment by sjinks** [comment]
I was not planning to merge it, as there are pending reviews from others teams.

**Comment by kat3samsin** [comment]
Closing this PR in lieu of this [PR](https://github.com/Automattic/vip-go-api-public/pull/1752) in GOOP.

**Pull request closed without merging**
